app/views/spam2/_nodes.html.erb

```diff
@@ -2,11 +2,7 @@
 $(document).ready(function () {
 	var table = table_main("#node_table");
 	$("#all").click(function () { //select all button
-		$('.selectedId').prop('checked', !$('.selectedId').prop('checked'));
-		var check = ($('.selectedId').filter(":checked").length == $('.selectedId').length);
-		$('#select-count').text($('.selectedId').filter(":checked").length);
-		$('#selectall').prop("checked", check);
-		disable_buttons('#selectall');
+              select_all();
```

Comment:
It looks like this file overall had tab indentation, which we actually don't prefer - we prefer 2-space indents. So you could do this:

```suggestion
    select_all();
```

We could fix the indentations in another issue, so that we don't make it harder to see what you've changed in this PR! I'll open a fto-candidate to suggest that!

Reply:
I did this in https://github.com/publiclab/plots2/issues/10952!